packages/headless/doc-parser/mocks/mock-func-entity.ts

```diff
@@ -8,7 +9,12 @@ export function buildMockFuncEntity(
     desc: '',
     name: '',
     params: [],
-    returnType: '',
+    returnType: buildMockEntity({
+      name: 'returnType',
+      type: 'void',
```

Comment:
I would prefer to use empty strings in the mocks. This will force the person writing the test to explicitly write out the expected values for these properties.

Reply:
I completely agree, but `returnType` no longer takes strings, only `AnyEntity` types. What would be the equivalent?

Reply:
Oh, I meant empty strings for the `name` and `type` params. I think it's possible to also just call `buildMockEntity` without any params.

```
returnType: buildMockEntity(),
```